docker/Real_device

```diff
@@ -156,7 +183,7 @@ EXPOSE 4723 6080 5555
 #===================
 # Run docker-appium
 #===================
-COPY sm/asm.jar /root/
+#COPY sm/asm.jar /root/
```

Comment:
I think we can delete the asm jar file that we dont need anymore.

Reply:
true!